source/_data/plugins.yml

```
@@ -2472,9 +2472,12 @@
     - steam
     - game
 - name: hexo-deployer-cos-cdn
-  description: Tencent COS deployment and CDN refresh plugin of Hexo | 将博客发布到腾讯COS并刷新CDN缓存
+  description: Tencent cos and Aliyun oss deployment and CDN refresh plugin of Hexo | 将博客发布到腾讯云、阿里云对象存储并刷新CDN缓存
```

```suggestion
  description: Deploy Hexo to OSS and purge the CDN cache
```

In case you are adding more CDN support for your plugin, I strongly suggest you update your description only for once.

---

Thank you for your comments. I have revised and submitted it.